dss/config.py

```diff
@@ -137,6 +140,25 @@ def get_s3_bucket() -> str:
 
         return Config._S3_BUCKET
 
+    @staticmethod
+    def get_s3_checkout_bucket() -> str:
+        if Config._S3_CHECKOUT_BUCKET is None:
+            if Config._CURRENT_CONFIG == BucketConfig.NORMAL:
+                envvar = "DSS_S3_CHECKOUT_BUCKET"
+            elif Config._CURRENT_CONFIG == BucketConfig.TEST:
+                envvar = "DSS_S3_CHECKOUT_BUCKET_TEST"
+            elif Config._CURRENT_CONFIG == BucketConfig.TEST_FIXTURE:
+                envvar = "DSS_S3_CHECKOUT_BUCKET_TEST_FIXTURES"
```

Comment:
i didn't see anything to populate this bucket.

Reply:
I'm planning to populate checkout fixture bucket in the near future